api/all/src/main/java/io/opentelemetry/api/trace/Span.java

```diff
@@ -75,6 +78,7 @@ static Span getInvalid() {
    */
   static Span wrap(SpanContext spanContext) {
     if (spanContext == null || !spanContext.isValid()) {
+      API_USAGE_LOGGER.finest("Span.wrap: context is null or invalid");
```

Comment:
Passing an invalid context is correct usage here, we need to split this if to only log for null